DEVEL.md

```diff
@@ -0,0 +1,78 @@
+# Developing the Maistra Istio Operator
+
+## Building the binary and the container image
+
+To build the Operator container image, run:
+```bash
+IMAGE=docker.io/maistra/istio-operator:latest tmp/build/docker_build.sh
+```
+
+This builds the binary, downloads and patches the Istio Helm charts, and finally builds the container image. The build script doesn't push the image to the container registry, so you'll need to push it manually:
+```bash
+docker push docker.io/maistra/istio-operator:latest
+```
+
+## Running the operator locally
+
+During development, you can run Istio Operator locally on your dev machine and have it reconcile objects in a local or remote OpenShift cluster. You can run the Operator from within your IDE and even use a debugger.
+
+To run the Operator locally, you need to set the following environment variables:
+
+- `KUBECONFIG=<path to your kubectl config file>`
+- `POD_NAME=istio-operator`
+- `POD_NAMESPACE=istio-operator`
```

Comment:
This is ignored by the operator; instead it reads it from a file:
`/var/run/secrets/kubernetes.io/serviceaccount/namespace`

Reply:
Ah, I've been using an extracted default token secret in `/var/run/secrets/...`, but when writing the doc, I realized that I can probably just set `KUBECONFIG` instead. I thought I removed the `/var/run/secrets/...` directory, but apparently not. 

Anyway, I've now submitted https://github.com/Maistra/istio-operator/pull/200 to allow us to run it without the namespace file, so `POD_NAMESPACE` now has to be set when running locally.